dashboard/lib/services/lti.rb

```diff
@@ -0,0 +1,27 @@
+require 'policies/lti'
+require 'user'
+require 'authentication_option'
+
+class Services::Lti
+  def self.partially_create_user(id_token)
```

Comment:
Consider renaming this `initialize_lti_user` since this would be similar to how we have `initialize_new_oauth_user` in other parts of the codebase. Maybe also provide a link to an example `id_token` if possible.

Reply:
✅